audit-service/README.md

```diff
@@ -18,8 +18,15 @@ limitations under the License.
 
 # Audit Service
 
-The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages contain all the details of the initial request and any other relevant information. This information will be passed to any local audit services that have
-been implemented. This service does not have any output Kafka topics.
+The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages contain all the details of the initial request and any other relevant information. 
+This information will be passed to any local audit services that have been implemented. This service does not have any output Kafka topics.
```

Comment:
Could do with more information on the local audit services.  As a developer what do I have to do to use this in my application?

Reply:
Added information about the interface at the start of the ReadMe